zappa/core.py

```diff
@@ -293,6 +292,8 @@ def __init__(self,
             # See https://github.com/pypa/manylinux for a more detailed explanation
             self.manylinux_wheel_file_suffix = 'cp38-manylinux1_x86_64.whl'
 
+        self.manylinux_wheel_abi3_file_match = re.compile('^.*cp3.-abi3-manylinux{1,2010}_x86_64.whl$')
```

Comment:
It looks like there is also a 2014 platform tag - see https://www.python.org/dev/peps/pep-0599/

Reply:
Yeah, and I checked and AWS Lambda supports it. Rebuilt the manylinux support so it works with all the 3 tags and added caching support for ABI3 ones. I think it's better now.